### DISCUSSION TITLE
Latest build download link

### DISCUSSION THREAD

**Discussion by protoxz** [Impostor/Impostor#367] [Q&A]
Hi !

Thanks for your Job !

Is there a way to get a download link to the linux server zip file latest build ?
thanks

**Comment by lbowers** [comment]
I think this is what you're looking for: https://ci.appveyor.com/project/Impostor/Impostor/build/artifacts

**Reply by protoxz** [reply]
Not exactly, i search a direct link download for example : .../latest-server-linux.zip
